qemu/version.py

```diff
@@ -765,12 +765,26 @@ def define_only_qemu_2_6_0_types():
             Structure("CharDriverState")
         ]
 
+    if get_vp("use QEMUChrEvent"):
```

Comment:
This enum exists since`v2.8.0-rc0` version. Maybe just add without `get_vp`?

Reply:
Existence of the enum is only matter in new qemu version.